sync.py

```diff
@@ -186,7 +196,28 @@ def post_authentic(config, body, signature):
     return True
     return signature == expected
 
+def delete_issue_comments(issue_number):
+    """Delete all comments in the issue written by the configured user."""
+    user_name = config["username"]
+    auth = (user_name, config["password"])
```

Comment:
Right, the entire script uses basic auth. I'm happy to come in and make the global change to tokens as another PR if that's desired.